uint32 obj_id
geometry_msgs/Point position
geometry_msgs/Vector3 velocity
float32[6] posvel_covariance_diagonal
